app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getGreeting } from '../features/greeting/greetingSlice';
+
+const Greeting = () => {
+  const { greeting } = useSelector((store) => store.greeting);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getGreeting());
+  }, []);
+
+  if (greeting.isLoading) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <div>
+      <h1>Random Message</h1>
+      <p>{greeting.body}</p>
+    </div>
+  )
+}
+
```

Comment:
- [ ] Good job so far, Kindly make sure that a random greeting is displayed on the frontend app, currently is not shown on your application 👍🏿